src/integrations/expressApollo.test.ts

```diff
@@ -25,6 +25,8 @@ function createApp(options: CreateAppOptions = {}) {
   return app;
 }
 
+function destroyApp(app) {}
```

Comment:
I think you should get rid of this and instead make the `destroyApp` argument optional. That way you only have to modify one existing test file.